Validation/RecoMuon/python/customiseForNewMuonValidation.py

```diff
@@ -0,0 +1,109 @@
+import FWCore.ParameterSet.Config as cms
+
+def enableNewMuonVal(process):
```

Comment:
again, see the previous comment. The old and new will coexist just for few prerelease cycles. The exact time we will take also depends on the proactivity of the offline managers, as we plan to validate the new code in 90X. (Hence we will need a backport)

Let me remind here that I have not proposed myself the overall plan, this has been proposed by the Muon POG conveners in agreement with relevant PPD coordinators as the safest way to be successfull (Gianluca Cerminara should remember). We have already spent a lot of time following this plan and waiting to be integrated in release. I think it should not be changed now.